app/src/org/commcare/activities/FormEntryActivity.java

```diff
@@ -1063,7 +1066,8 @@ private void loadForm() {
             }
 
             mFormLoaderTask = new FormLoaderTask<FormEntryActivity>(symetricKey, instanceIsReadOnly,
-                    formEntryRestoreSession.isRecording(), FormEntryInstanceState.mFormRecordPath, this, lastFormIndex) {
+                    formEntryRestoreSession.isRecording(), FormEntryInstanceState.mFormRecordPath, this, 
```

Comment:
this looks weird to have a comma just before method call close. No compilation errors here ?

Reply:
Noticed that too, bad conflict resolution. Redid it locally, pushing it in a few.